metallb/deploy_operator.sh

```diff
@@ -5,12 +5,7 @@ source ${metallb_dir}/metallb_common.sh
 
 METALLB_OPERATOR_REPO=${METALLB_OPERATOR_REPO:-"https://github.com/metallb/metallb-operator.git"}
 METALLB_OPERATOR_COMMIT=${METALLB_OPERATOR_COMMIT:-"4d52b86"}
-LOCAL_REGISTRY_DNS_NAME=${LOCAL_REGISTRY_DNS_NAME:-"virthost.ostest.test.metalkube.org"}
```

Comment:
these are removed because not necessary anymore?

Reply:
yes

Reply:
i going to remove them from the CI conf as well